### ISSUE TITLE
NFR Dash bar

### ISSUE THREAD

**Issue by NattBo** [colton-fjeldos/HackNSlash320#17]
The period of time where the player cannot dash will be represented by a bar at the bottom of the screen which fills up, with it being completely full representing the ability to dash again (accessibility)
